konan/konan.properties

```diff
@@ -68,7 +68,8 @@ cacheableTargets.macos_x64 = \
   ios_x64 \
   ios_arm64
 
-cacheableTargets.linux_x64 =
+cacheableTargets.linux_x64 = \
+  linux_x64
```

Comment:
Btw, did hidden visibility (`makeVisibilityHiddenLikeLlvmInternalizePass`) for static caches work as expected? I mean, does it help to make cached symbols non-exported in final executable or .so?

Reply:
Yes. More than that, it helps linker to cope with absence of some runtime declarations like `Kotlin_ObjCInterop_getTypeInfoForClass`.